handlers/practitioner_resource.go

```diff
@@ -55,6 +56,15 @@ func HandleCreatePractitionersResource(svc dao.Service) http.Handler {
 				return
 			}
 
+			// Check if 2 of the same practitioners were supplied in the same request
+			if _, exists := ipCodes[practitioner.IPCode]; exists {
```

Comment:
Would it work if the code were simplified like this?
`if ipCodes[practitioner.IPCode] {`

Reply:
Yes it would, good catch, I'll simplify this now